src/stores/SpectralProfileStore.ts

```diff
@@ -24,8 +24,14 @@ export class SpectralProfileStore {
         return null;
     };
 
-    @action clearProfiles = () => {
-        this.profiles = new Map<string, ObservableMap<CARTA.StatsType, ProcessedSpectralProfile>>();
+    public resetProfilesProgress = () => {
+        this.profiles.forEach(statsProfilesMap => {
```

Comment:
`this.profiles` is an observable, so shouldn't this still be an `@action` to avoid MobX warnings?